computedfields/graph.py

```diff
@@ -464,7 +464,9 @@ def resolve_dependencies(self, computed_models):
                                 path_segments.pop()
                         except FieldDoesNotExist:
                             # handle reverse relation (not a concrete field)
-                            rel = getattr(cls, symbol).rel
+                            descriptor = getattr(cls, symbol)
+                            # ReverseOneToOneDescriptor has a "related" attribute instead of "rel"
+                            rel = getattr(descriptor, "rel", None) or getattr(descriptor, "related")
```

Comment:
Oh dear, I was under the impression, that o2o reverse relations work the same as fk backrelations. This def. needs to be handled in the auto expansion as well to cover back updates. Also thought that is covered by tests already, dang.

Reply:
Where is auto expansion performed ? I added that change in `_extract_m2m_through` too, but didn't find any other place that needed it.

I made this change because otherwise it would crash when a Parent model would reference one of its Children models through the reverse O2O that's automatically created by Django in a muli-table inheritance setting, but I was really surprised that other O2O tests pass at all without this change, because I was under the impression that the O2O used for multi-table inheritance was no different from a "normal" O2O so I would have expected this issue to arise everytime a computed field depends on a reverse O2O, however this case seems already covered by the o2o tests and it worked correctly without those changes...

Reply:
The auto expansion is done in that loop, it adds backrelations for fk and m2m fields there. Need to test first whats going on with o2o relations, as they are meant to be tested in the o2o test cases (their back relation really should work). So for some reason, inherited o2o fields behave differently. This reason might be the fact, that inherited fields are mapped transparently onto the child model, but in fact live on a parent model. The deps resolver simply does catch that yet (again in that loop).

> I added that change in _extract_m2m_through too

Yes, thats basically the same deps resolver loop copied over, since we cannot pickle hidden through models (which is annoying for the fact it has to walk the deps a second time).